Python/Product/TestAdapter.Executor/Pytest/TestDiscovererPytest.cs

```diff
@@ -105,8 +105,9 @@ ITestCaseDiscoverySink discoverySink
                 foreach (PytestTest test in result.Tests) {
                     try {
                         (string parsedSource, int line) = test.ParseSourceAndLine();
-                        var parsedFullSourcePath = Path.IsPathRooted(parsedSource) ? parsedSource : Path.Combine(_settings.ProjectHome, parsedSource);
-                     
+                        // Note: we use _settings.ProjectHome and not result.root since it is being lowercased
+                        var parsedFullSourcePath = Path.IsPathRooted(parsedSource) ? parsedSource : PathUtils.GetAbsoluteFilePath(_settings.ProjectHome, parsedSource);
+
```

Comment:
also changed Path.Combine to PathUtils.GetAbsoluteFilePath, but its returning the same result